app/vue/package.json

```diff
@@ -43,15 +43,14 @@
     "find-cache-dir": "^1.0.0",
     "global": "^4.3.2",
     "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^2.30.1",
+    "html-webpack-plugin": "^3.0.7",
     "json5": "^0.5.1",
     "markdown-loader": "^2.0.2",
     "react": "^16.2.0",
-    "react-dev-utils": "^5.0.0",
+    "react-dev-utils": "zanettin/react-dev-utils",
```

Comment:
is this a repo we're waiting for changes to be merged and published to npm for `react-dev-utils`